docker/rust/Dockerfile

```diff
@@ -38,7 +38,10 @@ ONBUILD COPY target/debug/driver stablex
 FROM ${RUST_BASE}
 
 # Add Tini
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+# We ran into github rate limiting using this url so we now keep a local copy of
+# tini instead.
+# ENV TINI_VERSION v0.18.0
+# ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
```

Comment:
Not sure if it's worth keeping the old command around, but doesn't matter much.

Reply:
I think its nice to know where it came from.

Reply:
I would rather add this information to `docker/rust/README.md`

Reply:
I prefer it in this place because it right at the position where it is relevant so I'll leave it as is for now.